src/qutip_qip/circuit/circuit.py

```diff
@@ -879,16 +881,9 @@ def propagators(self, expand=True, ignore_measurement=False):
             )
         for gate in gates:
             if gate.name == "GLOBALPHASE":
-                qobj = gate.get_qobj(self.N)
-            else:
-                qobj = self._get_gate_unitary(gate)
-                if expand:
-                    all_targets = gate.get_all_qubits()
-                    qobj = expand_operator(
-                        qobj, dims=self.dims, targets=all_targets
-                    )
-            U_list.append(qobj)
-        return U_list
+                qobj = gate.get_ (self.N)
+                U_list.append(qobj)
+                continue
```

Comment:
I guess this is only temporary for debug?

Reply:
i think some rebase issue, fixed